### PR TITLE
Add Superblocks Lab to the Full IDE section

### PR DESCRIPTION
**Pull request by javier-tarazaga**
- Simply added Superblocks Lab to the Full IDE section

### CODE REVIEW

**Comment by styfle** [comment]
Hi Javier, thanks for the PR!

Superblocks Studio looks like a very unique IDE!

Please fix the description so that it complies with the requirements found in the [contributing.md](https://github.com/styfle/awesome-online-ide/blob/master/contributing.md) document, specifically number 7.

**Comment by javier-tarazaga** [comment]
Hi @styfle thanks! Sure thing, let me update the desc to make sure is compliant with the requirements.

File: README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ A curated list of awesome online implementations of integrated development envir
...
 * [MATLAB Online](https://www.mathworks.com/products/matlab-online.html) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run MATLAB code, visualize results, share scripts and collaborate on mathematical computing software
 * [Colaboratory](https://research.google.com/colaboratory/) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/googlecolab/colabtools) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run Python in a Jupyter notebook with ML libraries and free GPUs for running experiments
+* [Superblocks Studio](https://superblocks.com/studio) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/SuperblocksHQ/studio) - Superblocks studio is an open source online IDE developed for Ethreum DApp developers. Studio helps you code, build, and deploy your smart contracts for the Ethereum blockchain
```
Comment (styfle) [inline README.md]: The description should be 1 sentence, not 2.

The description should start with the word "Run".

Consider this:

> Run Ethereum DApps with local Ethereum blockchain debugging and deployment

**Comment by styfle** [comment]
Looks good, thanks 👍

**Pull request merged**